services/skipchain/msgs.go

```diff
@@ -0,0 +1,119 @@
+package skipchain
+
+import (
+	"github.com/dedis/cothority/network"
+	"github.com/satori/go.uuid"
+)
+
+func init() {
+	var msgs = []interface{}{
+		// Propose
+		&ProposeSkipBlock{},
+		&ProposedSkipBlockReply{},
+		&SetChildrenSkipBlock{},
+		&SetChildrenSkipBlockReply{},
+		// Propagation
+		&PropagateSkipBlock{},
+		// Requests for data
+		&GetUpdateChain{},
+		&GetUpdateChainReply{},
+		// Data-structures
+		&ForwardSignature{},
+		&SkipBlockFix{},
+		&SkipBlock{},
+	}
```

Comment:
Didn't we already had and reviewd that? I remember we coded that together before OSDI?